cmux_test.go

```diff
@@ -180,8 +220,7 @@ func TestClosed(t *testing.T) {
 	lis := mux.Match(Any()).(muxListener)
 	close(lis.donec)
 	mux.serve(closerConn{})
-	_, err := lis.Accept()
-	if _, ok := err.(errListenerClosed); !ok {
-		t.Errorf("expected errListenerClosed got %v", err)
+	if _, err := lis.Accept(); err != ErrListenerClosed {
```

Comment:
This error is a singleton, and only the instance is exported; this test provides documentation of the behaviour, but the old version does not.